src/Common/src/Interop/Windows/advapi32/Interop.EventRegister.cs

```diff
@@ -1,58 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Runtime.InteropServices;
-
-internal static partial class Interop
-{
-    internal static partial class Advapi32
```

Comment:
@brianrob, I could find any places where these tracing-related interop files were used.  Are they needed?

Reply:
I don't think these are needed.  We carry these alongside EventSource since it sinks between the various core repos.

Reply:
Ok, thanks.